app/models/campaign.js

```diff
@@ -18,6 +18,27 @@ var CampaignSchema = new Schema({
   }
 });
 
+// queries
+CampaignSchema.query.byTitleAndUser = function(title, influencerId) {
+    return Campaign.find({title: title, influencer: influencerId});
+};
+
+CampaignSchema.query.byUser = function(influencerId) {
+    return Campaign.find({influencer: influencerId});
+};
+
+// methods
+CampaignSchema.methods.delete = function(userId) {
```

Comment:
See the "Instance methods" section of the mongoose schema docs:
http://mongoosejs.com/docs/guide.html